djangocms_page_meta/templates/djangocms_page_meta/meta.html

```diff
@@ -36,17 +36,6 @@
 		{% if meta.twitter_author %}{% twitter_prop 'creator' meta.twitter_author %}{% endif %}
 		{% if meta.twitter_site %}{% twitter_prop 'site_name' meta.twitter_site %}{% endif %}
 	{% endif %}
-	{% if meta.use_googleplus %}
-		{% if meta.gplus_author %}
-			<link rel="author" href="{{ meta.gplus_author }}"/>
-		{% endif %}
-		{% if meta.title %}{% googleplus_prop 'name' meta.title %}{% endif %}
-		{% if meta.published_time %}{% googleplus_prop 'datePublished' meta.published_time %}{% endif %}
```

Comment:
I think we should keep valid `Schema.org` attributes

Reply:
according to schema.org documentation https://schema.org/WebPage all the properties currently in the template are available for a `WebPage` object:

- `name`
- `datePublished`
- `dateModified`
- `description`
- `url`
- `image`